proto/user.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+package examplepb;
+import "validate/validate/validate.proto";
+
+
+message CreateUserRequest{
+    string id = 1 [(validate.rules).string.uuid = true];
+    string first_name = 2 [(validate.rules).string.min_len = 1];
+    string last_name = 3 [(validate.rules).string.min_len = 1];
+    string user_name = 4 [(validate.rules).string.min_len = 1];
+    string email = 5 [(validate.rules).string.email = true];
+    string password = 6 [(validate.rules).string.min_len = 1];
+    string phone_number = 7;
+    google.protobuf.Timestamp date_of_birth = 8 [(validate.rules).timestamp.required = true];
+    int32 state_id = 9 [(validate.rules).fixed32 = {gte:1, lt: 38}];
+}
+
+    
+message CreateUserResponse{
+    string status = 1;
+}
+message UserMessage{
+    string id = 1;
+    string first_name = 2;
+    string last_name = 3;
+    string user_name = 4;
+    string email = 5; 
+    string password = 6;
```

Comment:
Remove this. You will not be passing the user password to any other service, it is to remain in the userService. You will also not be passing it to the frontend